source/extensions/filters/common/expr/context.cc

```diff
@@ -96,7 +96,7 @@ absl::optional<CelValue> ResponseWrapper::operator[](CelValue key) const {
       return CelValue::CreateInt64(code.value());
     }
   } else if (value == Size) {
-    return CelValue::CreateInt64(info_.bytesSent());
+    return CelValue::CreateUint64(info_.bytesSent());
```

Comment:
Can we leave this to be int64? It makes inter-op with other int64 values better. Otherwise, we have to cast in the expressions to int64, which is annoying.